x-pack/plugins/alerting/server/saved_objects/partially_update_rule.test.ts

```diff
@@ -39,14 +39,14 @@ describe('partially_update_rule', () => {
       });
 
       test('should work with extraneous attributes ', async () => {
-        const attributes = InvalidAttributes as unknown as PartiallyUpdateableRuleAttributes;
+        const attributes = ExtraneousAttributes as unknown as PartiallyUpdateableRuleAttributes;
         soClient.update.mockResolvedValueOnce(MockUpdateValue);
 
         await partiallyUpdateRule(soClient, MockRuleId, attributes);
         expect(soClient.update).toHaveBeenCalledWith(
           RULE_SAVED_OBJECT_TYPE,
           MockRuleId,
-          DefaultAttributes,
+          ExtraneousAttributes,
```

Comment:
@elastic/response-ops Now that we're defining AAD with an include list, any attribute not in the exclude list (or encrypted list) will be updatable without requiring re-encryption. This is a change in behavior - previously, extraneous attributes were filtered out because they were not in the exclude list. Is this ok? Might there be any unexpected side effects to consider?